packages/oidc-mock-provider/src/index.ts

```diff
@@ -87,6 +94,9 @@ export class OIDCMockProvider {
 
   private async init(): Promise<this> {
     this.httpServer.listen(this.config.port ?? 0, this.config.hostname);
+    if (this.config.bindIpAll) {
+      this.httpServer.listen(this.config.port ?? 0, '0.0.0.0');
```

Comment:
Should the two listen calls be mutually exclusive?

> The `server.listen()` method can be called again if and only if there was an error during the first `server.listen()` call or `server.close()` has been called

Reply:
Oh...